src/dg/dg_base.hpp

```diff
@@ -929,8 +932,29 @@ class DGBase
     bool use_auxiliary_eq;
     /// Set use_auxiliary_eq flag
     virtual void set_use_auxiliary_eq() = 0;
-}; // end of DGBase class
 
+public:
+    
+    /// Entropy FR correction at the current timestep
+    /** Used in entropy-RRK ODE solver.
+     * This is stored in dg such that both flow solver case and ode solver can access it. 
+     * flow solver cases have no access to ode solver. */
+    double FR_entropy_contribution=0;
+    
+    /// Entropy in the M norm
+    /** Rather than M+K norm, which is relevant for stabililty in FR.
+     * Used in entropy-RRK ODE solver.
+     * This is stored in dg such that both flow solver case and ode solver can access it. 
+     * flow solver cases have no access to ode solver. */
+    double entropy_M_norm=0;
+
+    /// Relaxation parameter
+    /** Used in RRK ODE solver.
+     * This is stored in dg such that both flow solver case and ode solver can access it. 
+     * flow solver cases have no access to ode solver. */
+    double relaxation_parameter=1;
```

Comment:
I think it's okay as-is -- to me, it makes sense that `relaxation_parameter` would be associated with `relaxation_runge_kutta`. 

Could another reviewer please weigh in on this? I can change it if there is consensus that it's confusing.

Reply:
Unless it's directly stored in a RRK class, relaxation_parameter applies to any solver. I see the comment saying it's stored in DG so that both flow solver and ode solver can access it. Is there a better way? Is RRK something that defines both time and space discretization? Otherwise, ideally, we would keep time-discretization parameters outside of DG.

Reply:
you could replace `flow_solver_case->compute_unsteady_data_and_write_to_table(ode_solver->current_iteration, ode_solver->current_time, dg, unsteady_data_table);` in `flow_solver.cpp` by `flow_solver_case->compute_unsteady_data_and_write_to_table(ode_solver, dg, unsteady_data_table);` which would be defined as a new virtual function in `FlowSolverCaseBase` with the default definition as just `flow_solver_case->compute_unsteady_data_and_write_to_table(ode_solver->current_iteration, ode_solver->current_time, dg, unsteady_data_table);`. Then in your flow solver case, you can override the definition for the new virtual function that takes the ode_solver as an argument and do whatever you need

Reply:
@jbrillon I like that idea, as it will allow everything to be contained in the ODE solver classes. However, I will need to do more restructuring than I have time for today, and other than this change I've addressed all comments. Therefore, is it okay if I make this change in a future PR which would only address this change? I will make sure to do it soon.

@PranshulThakur @dougshidong I have appended `_RRK_solver` to the variables I've added to DG. They would hopefully be removed in the future PR mentioned above.

Reply:
Sounds good. Thanks for making the change. I will leave it up to @jbrillon or @dougshidong to resolve this comment.

Reply:
If there's no rush for this PR then I'd say just address in this one when you have time